aries-hbase/src/main/java/org/apache/aries/PutWorker.java

```diff
@@ -64,6 +64,9 @@
   private final Parameter<Enum> value_kind =
       EnumParameter.newBuilder("pw.value_kind", VALUE_KIND.FIXED, VALUE_KIND.class)
                    .setDescription("Value is fixed or random generated").opt();
+  private final Parameter<Integer> key_length =
+      IntParameter.newBuilder("pw.key_length").setDefaultValue(Constants.DEFAULT_KEY_LENGTH_PW).
+          setDescription("How long will be the generated key.").opt();
```

Comment:
The description is not good. "The size of the key."

Reply:
Fixed in the latest commit.